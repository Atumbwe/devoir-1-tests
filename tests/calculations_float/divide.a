345 52.534 div