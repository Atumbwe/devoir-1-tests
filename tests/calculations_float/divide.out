 ___________________
< 6.567175543457571 >
 -------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
